docs/preview/features/correlation.md

```diff
@@ -7,21 +7,6 @@ layout: default
 
 The `Arcus.WebApi.Correlation` library provides a way to add correlation between HTTP requests. 
 
-- [Correlation Between HTTP Requests](#correlation-between-http-requests)
-  - [How This Works](#how-this-works)
-  - [Installation](#installation)
-  - [Usage](#usage)
-    - [Sending side](#sending-side)
-    - [Receiving side](#receiving-side)
-  - [Configuration](#configuration)
-    - [Configuring HTTP correlation services](#configuring-http-correlation-services)
-    - [Configuring HTTP correlation client tracking](#configuring-http-correlation-client-tracking)
-  - [Dependency injection](#dependency-injection)
-  - [Logging](#logging)
-  - [Using correlation within Azure Functions](#using-correlation-within-azure-functions)
-    - [Installation](#installation-1)
-    - [Usage](#usage-1)
```

Comment:
why was this part removed?

Reply:
Ah, since we already have this automatically generated in the right-side sidebar. All the titles and subtitles are already there so we don't have to duplicate it in the page itself.
LIke:
![image](https://user-images.githubusercontent.com/9039753/181242072-dbbdf094-b140-479d-95b4-a05962a9ec18.png)